accesscontroltool-bundle/pom.xml

```diff
@@ -121,6 +117,24 @@
             <artifactId>hamcrest-library</artifactId>
             <version>1.3</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.jackrabbit</groupId>
+            <artifactId>jackrabbit-jcr-commons</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>org.apache.sling.settings</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.day.jcr.vault</groupId>
+            <artifactId>com.day.jcr.vault</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
Please rather get rid of this dependency (even though currently only constants are referenced which are inlined anyways). It is fine to use those constants in the test classes (so leave it with scope test) but in the main classes we should not put this dependency (otherwise too easily a real runtime dependency towards Oak may slip in the code)